### PR TITLE
#166882308 Design the NotebooksPage

### PR DESCRIPTION
**Pull request by rethes**
**What does this PR do?**

This PR designs the NotebooksPage

**Description of Task to be completed?**

- add a NotebooksPage and NotebookTable components
- add styling

**How should this be manually tested?**

- git checkout `ft-design-notebook-page-166882308`
- yarn start
- navigate to your `localhost//notebooks`

**What are the relevant pivotal tracker stories?**

[#166882308](https://www.pivotaltracker.com/story/show/166882308)

**Screenshots (if appropriate)**

Screenshot 2019-06-27 at 15 36 26

Screenshot 2019-06-27 at 15 36 19

### CODE REVIEW

File: src/components/DropdownMenu/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+// third-party libraries
+import {Button, Dropdown, Header, Modal} from 'semantic-ui-react'
+
+// styles
+import './DropdownMenu.scss';
+
+//components
+
+const DropdownMenu = (props) => {
+  return (
```
Comment (micahoriaso) [inline src/components/DropdownMenu/index.js]: For function components, you can simplify them from 
```
const DropDownMenu = (props) => {
  return (
    <div>
        .
        .
        .
    </div>
  )
}
```
to 

```
const DropDownMenu = props => 
(
    <div>
        .
        .
        .
    </div>
 )
```

File: src/components/NotesMenu/index.js [hunk 2]
```diff
@@ -2,36 +2,33 @@
...
 // styles
 import './NotesMenu.scss';
 
-const NotesMenu = () => (
-  <div className="note-container">
-    <Menu secondary className="note-menu">
-      <Menu.Item name='Notes' icon='folder' className="note-item"/>
-    </Menu>
+//components
+import DropdownMenu from "../DropdownMenu";
```
Comment (micahoriaso) [inline src/components/NotesMenu/index.js]: Please be consistent in using quotes. If you are using singleqoutes for strings let it be uniform throughout the codebase.

File: src/components/DropdownMenu/index.js (hunk 1 above)
Comment (rethes) [reply src/components/DropdownMenu/index.js]: Done

File: src/components/NotesMenu/index.js (hunk 2 above)
Comment (rethes) [reply src/components/NotesMenu/index.js]: Noted

**Pull request merged**